packages/@aws-cdk/aws-apigatewayv2/lib/http/route.ts

```diff
@@ -167,7 +165,7 @@ export class HttpRoute extends Resource implements IHttpRoute {
       routeKey: props.routeKey.key,
       target: `integrations/${integration.integrationId}`,
       authorizerId: authBindResult?.authorizerId,
-      authorizationType,
+      authorizationType: authBindResult?.authorizationType ?? HttpAuthorizerType.NONE, // must be explicitly NONE (not undefined) for stack updates to work correctly
```

Comment:
Could you also update this test to assert that the default is set to NONE - https://github.com/aws/aws-cdk/blob/f0d5c25e1ae026eef03dc396e48368521dcb8331/packages/%40aws-cdk/aws-apigatewayv2/test/http/route.test.ts#L19